src/app/_components/RecentBlocks/RecentBlocks.tsx

```diff
@@ -0,0 +1,65 @@
+import { HStack, Icon } from '@chakra-ui/react';
+import { ArrowRight } from '@phosphor-icons/react';
+
+import { useGlobalContext } from '../../../common/context/useGlobalContext';
+import { buildUrl } from '../../../common/utils/buildUrl';
+import { Link } from '../../../ui/Link';
+import { TabsContent, TabsLabel, TabsList, TabsRoot, TabsTrigger } from '../../../ui/Tabs';
+import BitcoinIcon from '../../../ui/icons/BitcoinIcon';
+import StxIcon from '../../../ui/icons/StxIcon';
+import { RecentBtcBlocks } from './RecentBtcBlocks';
+
+export function RecentBlocks() {
+  const network = useGlobalContext().activeNetwork;
+  return (
+    <TabsRoot variant={'primary'} size={'redesignMd'} defaultValue={'btc'} ml={-3}>
```

Comment:
The ml={-3} seems to be compensating for the pl on the children elements. Why not just remove the pl?

Reply:
it's because we need the parent to have overflow hidden (to trim the list from right) but at the same time we want to show the shadow for the new block placeholder
<img width="206" alt="image" src="https://github.com/user-attachments/assets/8081f25a-e1e9-4814-ae6c-b3c5b1032b68" />